docs/configuration.rst

```diff
@@ -204,6 +204,12 @@ below:
        Example: ``DD_TRACE_PROPAGATION_STYLE_INJECT="datadog,b3"`` to inject both ``x-datadog-*`` and ``x-b3-*``
        headers into outbound requests.
 
+       .. _dd-trace-x-datadog-tags-max-length:
+   * - ``DD_TRACE_X_DATADOG_TAGS_MAX_LENGTH``
+     - Integer
+     - 128
```

Comment:
Should the default here be **512** instead of **128** (See comment in `config.py`)?